dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/dataelement/DefaultDataElementCategoryService.java

```diff
@@ -520,6 +526,9 @@ public void generateDefaultDimension()
         categoryCombo.addDataElementCategory( category );
         addDataElementCategoryCombo( categoryCombo );
 
+        categoryCombo.setPublicAccess( AccessStringHelper.CATEGORY_DEFAULT );
+        updateDataElementCategoryCombo( categoryCombo );
+

```

Comment:
CategoryCombo is not enable for data sharing. Check CategoryComboSchemaDescriptor, it doesnt have "schema.setDataShareable( true );"